src/App.tsx

```diff
@@ -1,7 +1,40 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handlerKeyUp);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handlerKeyUp);
+  }
+
+  handlerKeyUp = (event: KeyboardEvent) => {
+    this.setState({
+      pressedKey: event.code,
```

Comment:
you need `event.code` to check only some special keys (like `space`), in other cases - return `event.key`
<img width="226" alt="image" src="https://user-images.githubusercontent.com/55802630/183035470-ba5fd806-a361-499c-989e-fc7cfedcf990.png">